src/aiortc/contrib/media.py

```diff
@@ -164,10 +163,14 @@ def player_worker(
             try:
                 keypoints = keypoints_generator.get_keypoints(frame.to_rgb().to_ndarray())
                 keypoints_frame = KeypointsFrame(keypoints, frame.pts)
-                print("Keypoints for frame index %s retrieved." % str(frame.index))
+                logger.warning(
+                    "MediaPlayer(%s) Keypoints for frame index %s retrieved.", container.name, str(frame.index)
```

Comment:
Can't be sure about exact length of the line from this view, but please try to break this and the below line up so its not too long